File: Status_19_07_2022/pattern3.py
#!/usr/bin/python3

def pattern(row):
    i,j = 0,0
    while i < row:
        j = i+1
        while j:
            print(row-i,end = '')
            j -= 1
        print()
        i += 1
        
pattern(int(input("Enter the number of rows: ")))

File: Status_19_07_2022/pattern3.c
#include <stdio.h>

void pattern(int row)
{
    int i = 0, j = 0;
    while (i < row)
    {
        j = i+1;
        while (j--)
        {
            printf("%d",row-i);
        }
        printf("\n");
            i++;
    }
}

int main()
{
    int num = 0;
    printf("Enter the number of rows:\n");
    scanf("%d", &num);
    pattern(num);
}